File: csharp/Main/Friend.cs
using System;

namespace Soat.CleanCoders.DipKata.Main
{
    public class Friend
    {
        public string FirstName { get; }
        public string LastName { get; }
        public DateTime DateOfBirth { get; }
        public string Email { get; }

        public Friend(string firstName, string lastName, DateTime dateOfBirth, string email)
        {
            FirstName   = firstName;
            LastName    = lastName;
            DateOfBirth = dateOfBirth;
            Email       = email;
        }
    }
}

File: java/src/main/java/fr/soat/cleancoders/highlevel/Friend.java
package fr.soat.cleancoders.highlevel;

public class Friend {
    private final String name;
    private final String emailAdress;
    private final String phoneNumber;

    public Friend(String name, String email, String phoneNumber) {
        this.name = name;
        this.emailAdress = email;
        this.phoneNumber = phoneNumber;
    }

    public String getName() {
        return name;
    }

    public String getEmailAdress() {
        return emailAdress;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }
}
